src/components/Game/Answer.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import defaultPicture from '../../assets/images/profile_plaholder.png';
+export default class Answer extends Component {
```

Comment:
Using a stateless function component will be the good and more simple cause here we are not using state.

Reply:
@luc-tuyishime good ideal, I am working on it